client/hotrod-client/src/test/java/org/infinispan/client/hotrod/evolution/IndexSchemaNoDowntimeUpgradeTest.java

```diff
@@ -363,147 +359,6 @@ void testAddNonAnalyzedIndexOnExistingField() {
-        assertThat(doQuery("SELECT e.name, COUNT(e.name) FROM evolution.Model e WHERE e.name LIKE '%model%' GROUP BY e.name", cache))
-                .containsExactlyInAnyOrder(expected[0], expected[1], expected[2], expected[3], expected[4], expected[5]);
-
-        // projectable = true
-        assertThat(doQuery("SELECT e.name FROM evolution.Model e", cache)
-                .map(o -> ((Object[]) o)[0])).containsExactlyInAnyOrder("modelK # 0", "modelK # 1", "modelK # 2",
-                "modelL # 0", "modelL # 1", "modelL # 2");
-
-        // sortable = true
-        assertThat(doQuery("SELECT e.name FROM evolution.Model e ORDER BY e.name", cache).
-                map(o -> ((Object[]) o)[0])).containsExactly("modelK # 0", "modelK # 1", "modelK # 2",
-                "modelL # 0", "modelL # 1", "modelL # 2");
-    }
-
-    @Test
-    void testMigrateNormalizerAnalyzer() {
-        // VERSION 1
-        updateSchemaIndex(BaseModelIndexAttributesEntity.BaseModelIndexAttributesEntitySchema.INSTANCE);
-        RemoteCache<String, Model> cache = remoteCacheManager.getCache(CACHE_NAME);
-
-        // Create VERSION 1 entities
-        ModelUtils.createModelEntities(cache, 5, ModelUtils.createBaseModelIndexAttributesEntity(1));
-
-        // VERSION 1
-        // lowercase normalizer
-        doQuery("FROM evolution.Model e WHERE e.normalizedField LIKE '%normalized%'", cache, 5);
-        // standard analyzer => nothing found because standard analyzer takes whitespace as a delimiter when creating tokens
-        doQuery("FROM evolution.Model e WHERE e.analyzedField : '*analyzed field*'", cache, 0);
-
-        // Change attributes on indexes
-        updateSchemaIndex(BaseModelIndexAttributesChangedEntity.BaseModelIndexAttributesChangedEntitySchema.INSTANCE);
-
-        // Create VERSION 2 entities
-        ModelUtils.createModelEntities(cache, 5, ModelUtils.createBaseModelIndexAttributesChangedEntity(2));
-
-        // VERSION 2 - without reindex only newly added entries are recognized (5)
-        // lowercase normalizer removed => the query is case-sensitive, only newly added entries are found
-        doQuery("FROM evolution.Model e WHERE e.normalizedField LIKE '%NORMALIZED%'", cache, 5);
-        // standard analyzer => case-sensitive, whole text as one token, only newly added entries are found
-        doQuery("FROM evolution.Model e WHERE e.analyzedField : '*ANALYZED field*'", cache, 5);
-
-        remoteCacheManager.administration().reindexCache(CACHE_NAME);
-
-        // VERSION 2 - after reindex all entries are recognized (10)
-        // lowercase normalizer removed => the query is case-sensitive, also old entries are found
-        doQuery("FROM evolution.Model e WHERE e.normalizedField LIKE '%NORMALIZED%'", cache, 10);
-        // keyword analyzer => case-sensitive, whole text as one token, also old entries are found
-        doQuery("FROM evolution.Model e WHERE e.analyzedField : '*ANALYZED field*'", cache, 10);
-    }
-
```

Comment:
Was something not working?

Reply:
https://infinispan.org/docs/stable/titles/query/query.html#update-index-schema_indexing-caches